recipes/chromium/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "chromium" %}
+{% set version = "588429" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/{{ version }}/chrome-linux.zip  # [linux]
```

Comment:
Can you build from source please?

Reply:
I just wanted to take a shortcut, but it doesn't seem to be that simple. I think building chromium from source is too hard, so I'll close this PR.